cli/types/login.d.ts

```diff
@@ -0,0 +1,13 @@
+export {}
```

Comment:
Maybe just add a comment to the top of the file to the effect of 'the Auth0 DefinitelyTyped typings don't include the `BrandingClient`, so we shim their definitions (through module augmentation) to ensure that things compile correctly'

Reply:
Good catch. I'll say exactly that. Thanks Sam.